modules/broker-cli-client/src/main/java/org/wso2/broker/client/resources/Exchange.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.broker.client.resources;
+
+/**
+ * Representation of exchange in the broker
+ */
+public class Exchange {
+
+    public static final String NAME = "name";
+    public static final String TYPE = "type";
+    public static final String DURABLE = "durable";
+
+    private String name;
+
+    private String type;
+
+    private boolean durable;
+
+    public Exchange(String name, String type, boolean durable) {
+        this.name = name;
+        this.type = type;
+        this.durable = durable;
+    }
+
+    public String getAsJsonString() {
+        StringBuilder jsonString = new StringBuilder();
+        jsonString.append("{");
+        jsonString.append(NAME + ":" + name + ",");
+        jsonString.append(TYPE + ":" + type + ",");
+        jsonString.append(DURABLE + ":" + durable);
+        jsonString.append("}");
+        return jsonString.toString();
```

Comment:
What about replacing this with 
```
        return "{"
                + NAME + ":" + name + ","
                + TYPE + ":" + type + ","
                + DURABLE + ":" + durable
                + "}";
```
Since we can replace this string builder with a single string concatenation.

Reply:
will do that